pytorch_lightning/callbacks/xla_stats_monitor.py

```diff
@@ -75,8 +75,8 @@ def on_train_start(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule")
 
         if isinstance(trainer.accelerator, TPUAccelerator):
             raise MisconfigurationException(
-                "You are using XLAStatsMonitor but are not running on TPU"
-                f" since `tpu_cores` attribute in Trainer is set to {trainer.tpu_cores}."
+                "You are using XLAStatsMonitor but are not running on TPU."
+                f" The accelerator type is set to {trainer.accelerator.name().upper()}."
```

Comment:
`accelerator.name()` this doesn't exist anyone Ref: #12180

Reply:
Good catch!

BTW I think L76 should be `if not isinstance(trainer.accelerator, TPUAccelerator)` instead, do you know why pytorch_lightning/callbacks/xla_stats_monitor.py was not included in TPU tests?

Reply:
It will be resolved by #11098